f2/src/locales/en.json

```diff
@@ -6,14 +6,13 @@
   "LocationAnonymousPage.intro": "We use your report to link your report to other reports in your area and to coordinate with local police services across Canada.",
   "LocationAnonymousPage.title": "Enter your location",
   "SkipLink.text": "Skip to main content",
-  "anonymousForm.title": "Would you like to stay anonymous?",
   "anonymousPage.intro": "This means we won't ask for your contact details and won't be able to follow up in the future.",
   "anonymousPage.nextButton": "Continue",
   "anonymousPage.nextPage": "Next: How did the incident start?",
-  "anonymousPage.no": "No",
-  "anonymousPage.title": "Report anonymously?",
-  "anonymousPage.warning": "Select yes if you want to report anonymously",
-  "anonymousPage.yes": "Yes",
+  "anonymousPage.no": "anonymousPage.no",
```

Comment:
If we delete the `<Trans id="anonymousPage.no" />` tag right under the `return` in `AnonymousInfoForm.js` then we can delete `anonymousPage.no` here and in the French file (actually the post-commit hook will delete it for us)

Reply:
fixed